packages/spring_boot_endpoints/manifest.yml

```
@@ -0,0 +1,84 @@
+format_version: 1.0.0
+name: spring_boot_endpoints
+title: Spring Boot Endpoints
+version: 0.0.1
```

Version `0.0.1` doesn't have a changelog entry, but version `0.1.0` does. Should this be like this instead?
```suggestion
version: 0.1.0
```

---

yes its updated. Thanks!